stl/inc/xmemory

```diff
@@ -1621,31 +1619,31 @@ namespace ranges {
         requires constructible_from<iter_value_t<_Out>, iter_rvalue_reference_t<_It>>
     uninitialized_move_result<_It, _Out> _Uninitialized_move_unchecked(
             _It _IFirst, _Se _ILast, _Out _OFirst, _OSe _OLast) {

```

Comment:
Consider removing the extra level of indent from here through 1646, removing the `//` on 1625, and simply wrapping this entire function definition in `// clang-format off`.

Reply:
Aha! This is another case where clang-format is confused by a requires-clause without parentheses. If I add them, it formats the rest of the function nicely (except that it omits a space and says `requires(constructible_from`, and still needs help wrapping the `if constexpr` variable template usage).

I'll do this in a followup PR, thanks.